flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/src/main/java/org/apache/flink/cdc/connectors/kafka/utils/JsonRowDataSerializationSchemaUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.cdc.connectors.kafka.utils;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ReadableConfig;
+import org.apache.flink.formats.common.TimestampFormat;
+import org.apache.flink.formats.json.JsonFormatOptions;
+import org.apache.flink.formats.json.JsonRowDataSerializationSchema;
+import org.apache.flink.table.types.logical.RowType;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+/** Utils for creating JsonRowDataSerializationSchema. */
+public class JsonRowDataSerializationSchemaUtils {
+
+    /**
+     * In flink>=1.20, the constructor of JsonRowDataSerializationSchema has 6 parameters, and in
+     * flink<1.20, the constructor of JsonRowDataSerializationSchema has 5 parameters.
```

Comment:
Add todo to remove this class after bump to 1.20

Reply:
If  bump to Flink 1.20, maybe still need to use this class to be compatible with Flink 1.19 ?
because the parameter lengths of the JsonRowDataSerializationSchema constructors of <=1.19 and 1.20 are different.

Reply:
Actually, bumping to Flink 1.20 means that we won't make any guarantee that it can work well in lower version. This can help us get rid of always referencing other classes through reflection.